acceptance_tests/features/rh_ui_cookie_selection.feature

```diff
@@ -0,0 +1,50 @@
+@UI
+Feature: testing the cookie selection functionality of RH UI
+
+    Scenario: The cookies banner is displayed
+        When the UAC entry page is displayed
+        Then the cookies banner is displayed
+
+    Scenario: The 'cookies' hyperlink on the cookies banner points to the cookies page
+        Given the UAC entry page is displayed
+        And the cookies banner is displayed
+        Then the 'cookies' hyperlink on the cookies banner points to en/cookies/
+
+    Scenario: The "View cookies" hyperlink on the cookies banner points to the cookies page
+        Given the UAC entry page is displayed
+        And the cookies banner is displayed
+        Then the 'View cookies' hyperlink points to en/cookies/
+
+    Scenario: The "change cookie preferneces" hyperlink points to the cookies page
+        Given the UAC entry page is displayed
+        And the cookies banner is displayed
+        When the user accepts the cookies on the cookies banner
+        Then the "change cookie preferences" hyperlink text points to en/cookies/
+
+    Scenario: Accepting the cookies via the cookies banner is reflected in page cookies
+        Given the UAC entry page is displayed
+        And the cookies banner is displayed
+        When the user accepts the cookies on the cookies banner
+        Then all optional cookies are set to On
+        
+    Scenario: Rejecting the cookies via the cookies banner is reflected in page cookies
+        Given the UAC entry page is displayed
+        And the cookies banner is displayed
+        When the user rejects the cookies on the cookies banner
+        Then all optional cookies are set to Off
```

Comment:
You could probably put these into a scenario outline if you wanted to. Not much different between the two and you could avoid the similar code